### ISSUE TITLE
Changing focus in split mode slow

### ISSUE THREAD

**Issue by projekter** [texstudio-org/texstudio#595]
When using the split mode (both horizontal and vertical), changing the focus from, say the upper to the lower code panel, triggers high CPU consumption and actually takes more than one second to complete.
Using the mouse wheel to change the active document tab of the defocused panel allows to change the document that is currently displayed without changing the focus, and this works without any delay. Only then selecting, typing or scrolling in the document triggers the change of focus and the delay.

### Environment
- TeXstudio: 2.12.16beta3 (present in all previous version that supported split documents)
- Qt: 5.12.1
- OS: Windows 10
- TeX distribution: miktex

**Comment by manudroid19** [comment]
I have just tested in the same environment (latest beta, win10, miktex) and it doesn't happen to me. My documents were 50 pages and 10 pages large. Could it be because of the size of your files??
![gif](https://media.giphy.com/media/L0NL2t1fuiQHgoPQ0w/giphy.gif)

**Comment by projekter** [comment]
I started a fresh session without any documents and tried it again. Up to 2 by 2, indeed this is quite fast. However, starting with the third document (2 by 3), the delay begins for me (but if you don't know, you probably wouldn't notice), then is clearly noticable for 3 by 3. To test that it is not due to the sizes, I created six completely empty documents, three open on the left, three on the right.
What I can observe: If I quickly click on the left half, on the right, on the left again, repeating this (approx. 3-4 changes of focus per second), the focus does not change any more, but stays in the current tab. I am unable to reproduce this for 2 by 2 documents.
Of course, this is in a way a ridiculous stress test, but usually, I don't have six, but say ten to fifteen open and they are not empty, so TeXstudio probably has to do a lot more in the background regarding document analysis.
